server/lomas_server/dp_queries/dp_libraries/opendp.py

```diff
@@ -41,15 +125,16 @@ def cost(self, query_json: OpenDPInp) -> tuple[float, float]:
             tuple[float, float]: The tuple of costs, the first value
                 is the epsilon cost, the second value is the delta value.
         """
-        opendp_pipe = reconstruct_measurement_pipeline(query_json.opendp_json)
+        metadata = self.private_dataset.get_metadata()
```

Comment:
maybe we could set metadata in an __init__, no ? as it used for query and cost

Reply:
no, ok it's a good reason

Reply:
i did it just now for smartnoise sql

Reply:
Oh wait, I was thinking about the client library. But I guess it makes sense here, I will add it in the init.